### PR TITLE
add brightness control

### PR DESCRIPTION
**Pull request by arthurcgusmao**
Added the option to adjust the brightness also for those who may be interested.

### CODE REVIEW

File: src/redshift-adjust [hunk 1]
```diff
@@ -42,10 +42,14 @@ EOF
 fi
 
 if [ ! -e "${file_temp}" ]; then
-    echo 6500 > "${file_temp}"
+    temperature=$(redshift -p | grep -Pzo "(?<=Color temperature: ).*")
```
Comment (maandree) [inline src/redshift-adjust]: `-P`, `-z`, and `-o` are not in grep(1posix), can you use `sed` instead?

File: src/redshift-adjust [hunk 2]
```diff
@@ -42,10 +42,14 @@ EOF
...
+    temperature=${temperature%?} # remove last char (K)
```
Comment (maandree) [inline src/redshift-adjust]: This should be combined with `sed` above, and the `K` should be explicit.

File: src/redshift-adjust [hunk 3]
```diff
@@ -42,10 +42,14 @@ EOF
...
+    echo $temperature > "${file_temp}"
 fi
 if [ ! -e "${file_bright}" ]; then
-    echo 1.0 > "${file_bright}"
+    brightness=$(redshift -p | grep -Pzo "(?<=Brightness: ).*")
```
Comment (maandree) [inline src/redshift-adjust]: As above.

File: src/redshift-adjust [hunk 4]
```diff
@@ -42,10 +42,14 @@ EOF
...
+    echo $brightness > "${file_bright}"
+    $(pkill redshift) # killing redshift prevents default config to override new config
```
Comment (maandree) [inline src/redshift-adjust]: This should just be `pkill '^redshift$'`.

Comment (arthurcgusmao) [reply src/redshift-adjust]: Thanks, your suggestion solved a bug I was experiencing that the screen would blink and the settings would not be applied in the first time I was running redshift-adjust and redshift was running in the background.

File: src/redshift-adjust (hunk 1 above)
Comment (arthurcgusmao) [reply src/redshift-adjust]: I am trying, but I cannot find how to get only the content of where the line is using only `sed`. Would
```shell-script
temperature=$(redshift -p | grep 'Brightness:' | sed -e 's/Brightness: \(.*\)/\1/')
```
suffice? If not, any idea?

File: src/redshift-adjust (hunk 2 above)
Comment (arthurcgusmao) [reply src/redshift-adjust]: Here, do you mean that I should remove the K explicitly by using, e.g.,
```shell-script
redshift -p | grep 'Color temperature:' | sed -e 's/Color temperature: \(.*\)K/\1/'
```
or that I should leave the K in the file?

File: src/redshift-adjust (hunk 1 above)
Comment (maandree) [reply src/redshift-adjust]: `temperature=$(redshift -p | sed -n 's/Brightness: \(.*\)/\1/p')`

**Comment by arthurcgusmao** [comment]
Thank you for your support @maandree. I have made all modifications by [this commit](https://github.com/arthurcgusmao/redshift-adjust/commit/a2a9f04458852ed459bd33682025c86a58a19efa) and I'll open a new pull request on that one.

**Pull request closed without merging**